tests/e2e/specFiles/addingWallets/general.spec.ts

```diff
@@ -1,29 +1,37 @@
 import { waitFor } from 'detox';
 
 import { expectToBeDisabled } from '../../assertions';
+import { isBeta } from '../../helpers/utils';
 import app from '../../pageObjects';
 import steps from '../../steps';
 
 describe('Adding wallet', () => {
   describe('General', () => {
+    beforeEach(async () => {
+      isBeta() && (await app.onboarding.betaVersionScreen.close());
```

Comment:
we still test beta version?

Reply:
IMO we have still Beta version in environments, if we want to get rid of that we should clean up the app as well

Reply:
Discussed on daily - beta version is still possible to build so tests should be able to handle it.